frontend/src/components/world/MazeGame.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import { Box, useToast, Button, RenderProps, SimpleGrid } from '@chakra-ui/react';
+
+export default function MazeGame(): JSX.Element {
```

Comment:
I would rename this from `MazeGame` to `MazeGameInvite` or something since this component explicitly will be used for invite UI stuff